_benchmark/commands/command-flags.md

```diff
@@ -0,0 +1,204 @@
+---
+layout: default
+title: Command flags
+nav_order: 51
+parent: Command reference
+---
+
+# Command flags
+OpenSearch Benchmark (OSB) uses command line flags to change OpenSearch Benchmark's behavior. Not all flags can be used with each command. To find out which flags are supported by a specific command, enter `opensearch-benchmark <command> --h`.
+
+All command flags are added to a command using the following syntax:
+
+```bash
+opensearch-benchmark <command> --<command-flag>
+```
+
+Flags that accept comma-seperated values such `--telemetry` can also accept a JSON array. This can be defined by passing in a file path ending in `.json` or inline as a JSON string.
+
+- Comma-seperated values: `opensearch-benchmark ... --test-procedure="ingest-only,search-aggregations"`
+- JSON file: `opensearch-benchmark ... --workload-params="params.json"`
+- JSON inline string: `opensearch-benchmark  ... --telemetry='["node-stats", "recovery-stats"]'`
+
+## workload-path
+
+Can be either a directory that contains a `workload.json` file or a `.json` file with an arbitrary name that contains a workload specification. `--workload-path` and `--workload-repository` as well as `--workload` are mutually exclusive.
```

Comment:
```suggestion
This can be either a directory that contains a `workload.json` file or a `.json` file with an arbitrary name that contains a workload specification. `--workload-path` and `--workload-repository` as well as `--workload` are mutually exclusive.
```